celery/utils/saferepr.py

```diff
@@ -153,15 +152,15 @@ def _format_chars(val, maxlen):
     if isinstance(val, bytes):  # pragma: no cover
         return _format_binary_bytes(val, maxlen)
     else:
-        return "'{0}'".format(truncate(val, maxlen).replace("'", "\\'"))
+        return "'{}'".format(truncate(val, maxlen).replace("'", "\\'"))
```

Comment:
f

Reply:
Does not work.
See https://stackoverflow.com/questions/44780357/how-to-use-newline-n-in-f-string-to-format-output-in-python-3-6 for why.